src/components/memberInfo.jsx

```diff
@@ -29,7 +29,7 @@ const MemberInfoCard = ({ member, customTerms }) => {
                 size='xs'
               >
                 {`View 
-                ${address?.toLowerCase() === member?.memberAddress && 'my'}
+                ${address?.toLowerCase() === member?.memberAddress ? 'my' : ""}
```

Comment:
Why not the unary in this case?

Reply:
Because it is in a string is was printing this to the DOM:

View False Profile